ci/cloudbuild/builds/lib/features.sh

```diff
@@ -59,13 +59,28 @@ function features::libraries() {
   printf "%s\n" "${feature_list[@]}" | sort -u
 }
 
+function features::_internal_extra() {
+  local list=(
+    experimental-bigquery_rest
+    experimental-storage_grpc
+    oauth2
+    opentelemetry
+  )
+  printf "%s\n" "${list[@]}"
+}
+
 function features::list_full() {
   local feature_list
   mapfile -t feature_list < <(features::libraries)
-  feature_list+=(opentelemetry experimental-storage_grpc grafeas)
```

Comment:
FYI: we are dropping `grafeas` which I am ok with, but it is a feature that does something when we `-DGOOGLE_CLOUD_CPP_ENABLE=grafeas`. :shrug:

and we no longer test it here:

https://github.com/googleapis/google-cloud-cpp/blob/631c00ec32419fb88e201952c98fd92323f42840/ci/cloudbuild/builds/cmake-single-feature.sh#L23

Reply:
I am fine with the tradeoff in slightly less testing (the feature is tested indirectly anyway) vs. more manageably CI scripts.